event-log-kafka-lookup/pom.xml

```diff
@@ -74,11 +74,20 @@
             <groupId>org.apache.tinkerpop</groupId>
             <artifactId>gremlin-driver</artifactId>
             <version>${tinkerPopDriver.version}</version>
+            <classifier>shaded</classifier>
```

Comment:
For this service, this resolution is somehow needed, otherwise I couldn't connect to JanusGraph, even though the ThingApi and discovery service don't need it.